base_order_by_related/readme/USAGE.rst

```diff
@@ -0,0 +1,46 @@
+In our case we use this for the case of ordering common Odoo models by a non-stored computed field.
+
+This is normally not possible, but in this case it can be hacked in by creating a view such as::
```

Comment:
I found your example actually a bit too detailed. Don't we have some non-stored related field in the base module that you can use to make your point, same for tests?